### ISSUE TITLE
azure-cognitiveservices-nspkg has not been converted to implicit namespaces (PEP 420)

### ISSUE THREAD

**Issue by glaubitz** [Azure/azure-sdk-for-python#4461]
I am currently working on updating the Python Azure SDK in openSUSE/SLE to the latest upstream version. This involves switching over the packaging to adopt to the move to implicit namespace packages (PEP 420).

This has worked fine with ```azure-nspkg``` and ```azure-mgmt-nspkg``` so far, but ```azure-cognitiveservices-nspkg``` still behaves like the old namespace packages compiles and installs the ```__init__.py``` files.

Thus, the RPM builds complains that the namespace directories are not empty as they should after switching to implicit namespace packages for Python 3.7:

```
[    6s] error: Installed (but unpackaged) file(s) found:
[    6s]    /usr/lib/python3.7/site-packages/azure/cognitiveservices/__init__.py
[    6s]    /usr/lib/python3.7/site-packages/azure/cognitiveservices/__pycache__/__init__.cpython-37.opt-1.pyc
[    6s]    /usr/lib/python3.7/site-packages/azure/cognitiveservices/__pycache__/__init__.cpython-37.pyc
[    6s] 
[    6s] 
[    6s] RPM build errors:
[    6s]     Installed (but unpackaged) file(s) found:
[    6s]    /usr/lib/python3.7/site-packages/azure/cognitiveservices/__init__.py
[    6s]    /usr/lib/python3.7/site-packages/azure/cognitiveservices/__pycache__/__init__.cpython-37.opt-1.pyc
[    6s]    /usr/lib/python3.7/site-packages/azure/cognitiveservices/__pycache__/__init__.cpython-37.pyc
```

**Comment by lmazuel** [comment]
Hi @glaubitz 
You're right, I'm on it. Thanks for the report!

**Comment by lmazuel** [comment]
All of them will be released in a few minutes as 3.0.1, please re-open an issue if I missed something about it.
Thanks!

**Comment by glaubitz** [comment]
Thanks. I'll pull the updates on Monday and see if I can update all packages then. If I find something, I'll let you know.
